### PR TITLE
Merge 2017-04-02

### PR DESCRIPTION
**Pull request by rwhogg**

### CODE REVIEW

File: screen.rb [hunk 1]
```diff
@@ -16,26 +16,25 @@ class Screen < Formula
...
-      url "https://gist.githubusercontent.com/yujinakayama/4608863/raw/75669072f227b82777df25f99ffd9657bd113847/gistfile1.diff"
-      sha256 "9c53320cbe3a24c8fb5d77cf701c47918b3fabe8d6f339a00cfdb59e11af0ad5"
+      url "https://gist.githubusercontent.com/sobrinho/5a7672e088868c2d036957dbe7825dd0/raw/c6fe5dc20cb7dbd0e23f9053aa3867fcbc01d983/diff.patch"
+      sha256 "47892633ccb137316a0532b034d0be81edc26fc72a6babca9761a1649bc67fd1"
     end
   end
 
   depends_on "autoconf" => :build
   depends_on "automake" => :build
-  depends_on "homebrew/dupes/ncurses" unless OS.mac?
```
Comment (rwhogg) [inline screen.rb]: Darn, I meant to keep that.

**Comment by rwhogg** [comment]
From https://travis-ci.org/Linuxbrew/homebrew-dupes/builds/217918502#L2295

```
/bin/bash: line 9: makeinfo: command not found
```

`bc` needs `texinfo` to to build.

From https://travis-ci.org/Linuxbrew/homebrew-dupes/builds/217918502#L9180

```
==> brew linkage --test fzf
Error: Failure while executing: ["patchelf", "--print-needed", "/home/linuxbrew/.linuxbrew/Cellar/fzf/0.16.6/bin/fzf"] 
```

A few things worth noting here @sjackman:

1. This is reproducible for me locally. The error is `cannot find section .dynamic`. Statically linked I guess?
2. Setting $HOMEBREW_ENV_FILTERING seems to break `brew linkage` in general (i.e. this `fzf` failure happens with or without it, but with that environment variable set it seems to break for other formulae too).

**Comment by rwhogg** [comment]
From the raw Travis logs: 

```
brew linkage --test scrollkeeper
Missing libraries:
  libxslt.so.1
```

Can confirm by testing locally in superenv:

```
checking for xslt-config... no
configure: error: Couldn't find xslt-config please install the libxslt package
```

**Comment by sjackman** [comment]
> This is reproducible for me locally. The error is cannot find section .dynamic. Statically linked I guess?

Fixed by https://github.com/Linuxbrew/brew/pull/324

**Pull request merged**